### PR TITLE
simplify the if/else logic

### PR DESCRIPTION
**Pull request by lewgun**
**What problem does this PR solve?**

simplify the if/else logic

**What is changed and how it works?**

refactor to simplify the if/else logic

**Check List**

Tests

- Unit test

Code changes

- N/A

Side effects

- N/A

Related changes

- N/A

### CODE REVIEW

File: pkg/diff/chunk.go [hunk 1]
```diff
@@ -243,25 +243,26 @@ func (s *randomSpliter) splitRange(db *sql.DB, chunk *chunkRange, count int, sch
 		symbolMax = chunk.bounds[colNum-1].upperSymbol
 	} else {
 		// choose the next column to split data
```
Comment (WangXiangUSTC) [inline pkg/diff/chunk.go]: please move this comment to L250

**Review by WangXiangUSTC — COMMENTED** [review]
thanks! rest LGTM

File: pkg/diff/chunk.go (hunk 1 above)
Comment (lewgun) [reply pkg/diff/chunk.go]: done PTAL @WangXiangUSTC

**Comment by WangXiangUSTC** [comment]
LGTM

**Pull request merged**